### DISCUSSION TITLE
Kimai 1 - some entries disappear

### DISCUSSION THREAD

**Discussion by Ninister** [kimai/kimai#5027] [General]
Hello, I still use Kimai 1 about 10 years now.
But I got this problem, that some entries disappear and don't show up In the list. The list shows entries from 1.1.2023 until today.
Today I made two entries, the first is visible, the second not. I checked the database, the entry is in the database and I can't see differences in the database, why the second does not show up.
![image](https://github.com/user-attachments/assets/eb761750-dea5-47f4-8bfd-c95c81a8ba26)

**Comment by kevinpapst** [comment]
No support given for Kimai 1, which is not maintained since many years.

You should upgrade to Kimai 2.

**Comment by Ninister** [comment]
thanks, I found it. Somehow I set a date in the future. I made a mistake, not a problem of Kimai.
I know about Kimai 2, but my system runs and I only need the functions of v1.
